go.mod

```diff
@@ -1,11 +1,11 @@
-module github.com/gamalan/caddy-tlsredis
+module github.com/regbo/caddy-tlsredis
```

Comment:
I don't code go, just curious, is this intended?

Reply:
I don't either, first time using it and not intentional. I had to make this change to use it with xcaddy. Definitely remove when merging.